lambda/custom/helperFunctions.js

```diff
@@ -1,9 +1,9 @@
-
 const axios = require('axios');
 const apiEndpoints = require('./apiEndpoints');
 const envVariables = require('./config');
 
-
+const Jargon = require('@jargon/alexa-skill-sdk')
+const ri = Jargon.ri
```

Comment:
Just a question. In this file, how can JARGON know what language it has to use?

Reply:
That's a really good question.
And I am not completely sure of the answer, but I have messaged Jargon team regarding this and we should find out soon.

Reply:
BTW I tested it using italian and creating a new channel with 2 words (for obtaining an error). It worked anyways so I think they share the same instance allowing them to know what language is in use.

Reply:
Awesome. Then we can merge this one @Sing-Li